test/powershell/Language/Scripting/ScriptHelp.Tests.ps1

```diff
@@ -88,34 +88,34 @@ Describe 'get-help HelpFunc1' -Tags "Feature" {
     }
 
     Context 'get-help helpFunc1 -component blah' {
-        $x = get-help helpFunc1 -component blah -ea SilentlyContinue -ev e
+        $x = get-help helpFunc1 -component blah -ErrorAction SilentlyContinue -ev e
```

Comment:
Please expand `-ev`. Multiple instances in this file.

Reply:
Fixed.